ui/app/AppLayouts/Chat/views/ContactsColumnView.qml

```diff
@@ -104,12 +104,13 @@ Item {
         }
 
         // search field
-        StatusBaseInput {
+        StatusInput {
             id: searchInput
             Layout.fillWidth: true
             Layout.preferredHeight: 36
-            placeholderText: qsTr("Search")
-            icon.name: "search"
+            maximumHeight: 36
+            input.placeholderText: qsTr("Search")
```

Comment:
On a second thought, this could be a forwarding alias... wdyt? So you'd still be able to say `placeholderText: qsTr("Search")`

Reply:
done as well :)